index.html

```diff
@@ -0,0 +1,471 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src='main.js'></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body onload="modalDisplay()">
+    <div class="nav_modal-background">
+      <div class="nav_modal">
+        <div class="nav_modal-header">
+          <h2 class="nav_modal-title"></h2>
+          <i class="fa fa-times service-icon X"></i>
+        </div>
+        <div class="nav_modal-body">
+            <ul class="nav_ul">
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#projects">Portfolio</a>
+              </li>
+              <li class="divider"></li>
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#about-me">About me</a>
+              </li>
+              <li class="divider"></li>
+              <li class="nav_li">
+                <a class="modal_nav_link" href="#contact">Contact me</a>
+              </li>
+              <li class="divider"></li>
+            </ul>
+        </div>
+      </div>
+    </div>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars" id="nav_modal-displayer"></i>
+      </button>
```

Comment:
I suggest making the button (hamburger button) inside the <nav> tag because the button is also part of the navigation on this page, and that way it will the button makes sense inside the <nav> tag